### PR TITLE
feat:add short_name column and update description in sync types

### PR DESCRIPTION
**Pull request by ramiroJCB**
Here we me make basic migration on sync_types  and also documenting the  migration  process

### CODE REVIEW

File: migrations/20220831010237_sync_types_descriptions.up.sql [hunk 1]
```diff
@@ -0,0 +1,56 @@
+BEGIN;
+
+ALTER TABLE mergestat.repo_sync_types
+ADD COLUMN short_name varchar(255) NOT NULL DEFAULT '';
```
Comment (patrickdevivo) [inline migrations/20220831010237_sync_types_descriptions.up.sql:4]:
```suggestion
ADD COLUMN short_name text NOT NULL DEFAULT '';
```

File: migrations/20220831010237_sync_types_descriptions.up.sql [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        THEN 'GitHub Repo Metadata' 
+        WHEN TYPE = 'GITHUB_REPO_PRS' 
+        THEN 'GitHub Repo Pull Requests' 
+        WHEN TYPE = 'GITHUB_REPO_STARS' 
+        THEN 'GitHub Repo Stars' 
+        WHEN TYPE = 'GIT_REFS' 
+        THEN 'Git Refs' 
+    END );
+
+COMMIT;
```
Comment (patrickdevivo) [inline migrations/20220831010237_sync_types_descriptions.up.sql:56]:
```suggestion
COMMIT;

```

File: migrations/20220831010237_sync_types_descriptions.up.sql (hunk 1 above)
Comment (ramiroJCB) [reply migrations/20220831010237_sync_types_descriptions.up.sql]: done :)

File: migrations/20220831010237_sync_types_descriptions.up.sql (hunk 2 above)
Comment (ramiroJCB) [reply migrations/20220831010237_sync_types_descriptions.up.sql]: for some reason it didn't added my  local ref to this 
Screen Shot 2022-08-31 at 9 38 49 AM

**Comment by ramiroJCB** [comment]
@patrickdevivo   seems good now

**Pull request merged**